gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -99,6 +103,13 @@ public final class InstantiatingGrpcChannelProvider implements TransportChannelP
   @VisibleForTesting
   static final String DIRECT_PATH_ENV_ENABLE_XDS = "GOOGLE_CLOUD_ENABLE_DIRECT_PATH_XDS";
 
+  // The public portion of the mTLS MDS root certificate is stored for performing
+  // cert verification when establishing an mTLS connection with the MDS.
+  private static final String MTLS_MDS_ROOT = "/run/google-mds-mtls/root.crt";
+  // The mTLS MDS credentials are formatted as the concatenation of a PEM-encoded certificate chain
+  // followed by a PEM-encoded private key.
+  private static final String MTLS_MDS_CERT_CHAIN_AND_KEY = "/run/google-mds-mtls/client.key";
```

Comment:
nit: Rename this to something like `MTLS_MDS_CERT_CHAIN_AND_KEY_PATH` or `MTLS_MDS_CERT_CHAIN_AND_KEY_LOCATION`, since they are just paths to the key, not the content of the key. Same thing for the above `MTLS_MDS_ROOT`.

Reply:
Done in https://github.com/googleapis/sdk-platform-java/pull/3326/commits/c9a7edd268003058264b9dc4b079aad5d01582c5